src/unity.yml

```diff
@@ -49,16 +45,22 @@ paths:
     $ref: './unity/paths/orgs_orgId_users_userId.yml'
   '/orgs/{orgId}/limits':
     $ref: './unity/paths/orgs_orgId_limits.yml'
-  '/orgs/{orgId}':
-    $ref: './unity/paths/orgs_orgId.yml'
```

Comment:
this route is unused and kind of a dupe of the `GET /operator/orgs/{orgId}` route